modules/server_processes_manager/php/serverprocesslauncher.class.inc

```diff
@@ -70,7 +70,7 @@ class ServerProcessLauncher
      *
      * @param AbstractServerProcess $process the process that was just started
      *
-     * @return void
+     * @return string
```

Comment:
Agreed, however this type change should probably be done in the Database class. The function `lastInsertID` should be changed to return an int instead of a string.